src/client/pages/OfferNew/AppPromotionSection.tsx

```diff
@@ -0,0 +1,44 @@
+import styled from '@emotion/styled'
+import { colorsV3 } from '@hedviginsurance/brand'
+import React from 'react'
+import { useTextKeys } from 'utils/textKeys'
+import { LARGE_SCREEN_MEDIA_QUERY } from 'utils/mediaQueries'
+import { LocalizedImage } from 'components/LocalizedImage'
+import { Column, ColumnSpacing, Container, Heading, Body } from './components'
+
+const OuterWrapper = styled.div`
+  padding: 5rem 0;
+  background-color: ${colorsV3.gray100};
+
+  ${LARGE_SCREEN_MEDIA_QUERY} {
+    padding: 10rem 0;
+  }
+`
+
+const PromotionImage = styled(LocalizedImage)`
+  margin-top: 1.5rem;
+
+  ${LARGE_SCREEN_MEDIA_QUERY} {
+    margin-top: 2.5rem;
+  }
+`
+
+export const AppPromotionSection: React.FC = () => {
+  const textKeys = useTextKeys()
+
+  return (
+    <OuterWrapper>
+      <Container>
+        <Column>
+          <Heading>{textKeys.OFFER_APP_PROMOTION_HEADING()}</Heading>
+          <Body>{textKeys.OFFER_APP_PROMOTION_PARAGRAPH()}</Body>
```

Comment:
I assume this will have updated color and size once you're other PR is merged :)

Reply:
Yes exactly, I will merge that first and have a look so this is in good shape